src/views/Give/RedeemYield.tsx

```diff
@@ -56,12 +64,15 @@ export default function RedeemYield() {
     return state.pendingTransactions;
   });
 
-  const redeemableBalanceNumber: BigNumber = new BigNumber(redeemableBalance);
-
-  const totalDeposit = new BigNumber(recipientInfo && recipientInfo.totalDebt ? recipientInfo.totalDebt : 0);
+  const totalDeposit = new DecimalBigNumber(
+    recipientInfo && recipientInfo.totalDebt ? recipientInfo.totalDebt : "0",
+    9,
+  );
 
   const stakingRebasePercentage = new BigNumber(stakingRebase ? stakingRebase : 0).multipliedBy(100);
-  const nextRewardValue = new BigNumber(stakingRebase ? stakingRebase : 0).multipliedBy(totalDeposit);
+  const nextRewardValue = totalDeposit
+    .mul(new DecimalBigNumber(stakingRebase ? stakingRebase.toString() : "0", 2))
```

Comment:
Does this multiplication look right or should I use stakingRebasePercentage instead?

Reply:
I fixed it in the react-query branch, so when we merge that, it'll be fixed.